Yoast/wordpress-seo
# Add the translations of the javascript monorepo packages

## Summary

I presume that since we moved to the javascript monorepo there are some translations missing.
This adds the packages that actually have translations via `@wordpress/i18n` to the pipeline of `yoast-components`.
The textdomain is because that is how they are specified in the JS code. Perhaps later we should split this up properly.


This PR can be summarized in the following changelog entry:

* Fixes a bug where certain translations would be missing in the metabox, sidebar, configuration wizard and the help center.

## Relevant technical choices:

* See summary.

## Test instructions

This PR can be tested by following these steps:

* Run `grunt clean:language-files`, which empties the `languages` directory.
* Run `grunt build:i18n:potFiles` to read the translations used in the code.
* Check that there are 6 `yoast-js-*.pot` files. These each contain the JS translations of the corresponding packages.
* The issue itself is about the `analysis-report` package having the strings: `Problems`, `Improvements` and `Good results`. Check that the `yoast-js-analysis-report.pot` file has these strings.
* Then those 6 files should already be combined into the `yoast-components.pot` file. So double check that they are in there too.
* Run `grunt build:i18n:translations` to fetch the translations from GlotPress and process them accordingly.
* The same translations as before should now live in the `yoast-components.php` file.
* The `yoast-components-{LOCALE}.json` files are the ones that contain the translations for use in JavaScript. However, they do not contain untranslated strings. In the case of this issue it seems the strings already exist though, so can be seen in there too: `Problems`, `Improvements` and `Good results`.
* The already translated ones can be checked in the plugin. For the ones in the issue go to a post and check the SEO and/or Readability results.

## UI changes

## Documentation

## Quality assurance


Fixes #13344
